camera_calibration/scripts/tarfile_calibration.py

```diff
@@ -226,4 +229,15 @@ def cal_from_tarfile(boards, tarname, mono = False, upload = False, calib_flags
     if (num_ks < 1):
         calib_flags |= cv2.CALIB_FIX_K1
 
-    cal_from_tarfile(boards, tarname, options.mono, options.upload, calib_flags, options.visualize, options.alpha)
+    pattern = None
+    if options.pattern == "chessboard":
+        pattern = Patterns.Chessboard
+    if options.pattern == "circles":
+        pattern = Patterns.Circles
+    if options.pattern == "acircles":
+        pattern = Patterns.ACircles
+    if pattern is None:
+        parser.error("Must specify pattern type")
```

Comment:
Since this is a new field, it should not be required. Please default this to "chessboard" (which was likely the only option before).

Reply:
updated